rotkehlchen/tasks/calendar.py

```diff
@@ -195,3 +197,98 @@ def maybe_create_ens_reminders(database: DBHandler) -> None:
+            ),
+        ))
+
+    if len(crv_events) == 0:
+        return
+
+    db_calendar = DBCalendar(database=database)
+    customizable_date = CustomizableDateMixin(database=database)
+    with database.conn.read_ctx() as cursor:
+        blockchain_accounts = database.get_blockchain_accounts(cursor=cursor)
+
+    current_ts = ts_now()
+    crv_calender_entries: list[int] = []
+    for crv_event in crv_events:
+        if not (
+            crv_event.location_label is not None and
+            crv_event.extra_data is not None and
+            (locktime := crv_event.extra_data.get('locktime')) is not None and
+            (amount := crv_event.balance.amount) is not None and
+            (user_address := string_to_evm_address(crv_event.location_label)) in blockchain_accounts.get(  # noqa: E501
+                blockchain := ChainID.deserialize(crv_event.location.to_chain_id()).to_blockchain(),  # noqa: E501
+            ) and (locktime := Timestamp(locktime)) > current_ts  # those ending in the future
+        ):
+            continue
+
+        calendar_entry_description = f'Lock period for {amount} CRV in vote escrow ends on {customizable_date.timestamp_to_date(locktime)}'  # noqa: E501
+        if (db_calendar.query_calendar_entry(
+                filter_query=CalendarFilterQuery.make(
+                    and_op=True,
+                    name='CRV vote escrow lock period ends',
+                    addresses=[OptionalBlockchainAddress(
+                        blockchain=blockchain,
+                        address=user_address,
+                    )],
+                    blockchain=blockchain,
+                    counterparty=CPT_CURVE,
+                    description=calendar_entry_description,
+                ),
+        ))['entries_found'] == 0:  # if calendar entry doesn't exist, add it
+            crv_calender_entries.append(db_calendar.create_calendar_entry(CalendarEntry(
+                name='CRV vote escrow lock period ends',
+                timestamp=locktime,
+                description=calendar_entry_description,
+                counterparty=CPT_CURVE,
+                address=user_address,
+                blockchain=blockchain,
+                color=CRV_CALENDAR_COLOR,
+                auto_delete=True,
+            )))
+        else:
+            continue  # skip if it already exists
+
+    _, failed_to_add = db_calendar.create_reminder_entries(reminders=[
+        ReminderEntry(
+            identifier=calendar_identifier,  # this is only used for logging below, it's auto generated in db  # noqa: E501
+            event_id=calendar_identifier,
+            secs_before=secs_before,
+        )
+        for calendar_identifier in crv_calender_entries
+        for secs_before in (WEEK_IN_SECONDS, DAY_IN_SECONDS)
+    ])
+
+    with database.conn.write_ctx() as write_cursor:
+        database.set_static_cache(
+            write_cursor=write_cursor,
+            name=DBCacheStatic.LAST_CREATE_REMINDER_CHECK_TS,
```

Comment:
we set this cache in the other functions also where we set reminders, and this can interfere with them. I'd set it in a common function after all these reminders functions are executed